lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -76,7 +76,7 @@ def override_base_path(&base_path_builder)
     def perform(output: true)
       scope.find_each do |whitehall_model|
         queue_check("content-store", whitehall_model, output)
-        queue_check("draft-content-store", whitehall_model, output)
+        queue_check("draft-content-store", whitehall_model, output) if whitehall_model.draft?
```

Comment:
Not sure about this, but I kept getting models that aren't drafts being compared to published versions from the `draft-content-store` which have things like updated titles and what-not.